.travis.yml

```diff
@@ -1,14 +1,14 @@
+sudo: required
```

Comment:
Hardcoding the version however might actually give you less trouble in the future if you just want "passive support" for JRuby. If you hardcode it, the JRuby version in your CI environment will be unchanged, so you're guaranteed that future updates on the environment won't break your build.

Of course you can also see this "variability" as a good thing since it can help you detect bugs and track compatibility but given the type of JRuby support you describe, you might actually want to lock the version and leave it up to actual JRuby users to keep it up to date via contributions.

NOTE: This is all assuming that obviously changes in your own code (or your dependencies, since you don't commit your `Gemfie.lock` to source control) can always break your `JRuby` build, but that can happen against both hardcoded or non hardcoded version).